apps/barista/src/components/live-example/live-example.ts

```diff
@@ -111,33 +132,89 @@ export class BaLiveExample implements OnDestroy {
    * @internal
    * The currently active tab for displaying a source (html, ts or scss).
    */
-  _activeTab: 'html' | 'ts' | 'scss';
+  _activeTab: BaSourceType;
+
+  /** @internal Whether the source code is hidden. */
+  _sourceHidden = false;
+
+  /** @internal Whether the source code is expanded to its max height. */
+  _isExpanded = false;
+
+  /** @internal Whether the source code has been copied. */
+  _copied$: BehaviorSubject<boolean> = new BehaviorSubject(false);
```

Comment:
In your case having an observable is not the necessary as it is just reflecting a state without any transformation or additional stuff. Having just a property is fine.